.github/workflows/release-conda.yml

```diff
@@ -3,7 +3,7 @@ name: Release - Conda
 on:
   push:
     tags:
-      - v*
+      - '*'
```

Comment:
Let's try it like this then! Should trigger on all tags.

Reply:
What about `[0-9]+.[0-9]+.[0-9]+*` ?

Reply:
Sure, just pushed it!